### ISSUE TITLE
[Feature Request] Influence on accuracy

### ISSUE THREAD

**Issue by ErfolgreichCharismatisch** [readbeyond/aeneas#271]
I used the web app for aligning.

I found that 54 % of phrases in my test set were misaligned.

Misaligned meaning

1. Cut too early from the end
2. Cut too late from the previous fragment
3. Shifted altogether

I had voice detection control whether the aligned parts matched(aeneas -> cue list -> cut audio -> have voice recognition detect speech -> compare to entries in cue list with a similarity algorithm).

Now, more than half misaligned is discouraging.

Currently I did not see many options to influence recognition, the parameters of the cli seem rather cosmetic in influence, barring language selection and input text type.

I would like to improve alignment, have a confidence message to be able to quickly review or discard.

The pipeline contains everything required for a confidence parameter. Also other parameters for deep control are important.

What I am deeply missing is a threshold parameter in decibel to define pauses and audio - this would eliminate premature cuts for good.

**Comment by versae** [comment]
Did you find any good solution to the misalignments?

**Comment by ErfolgreichCharismatisch** [comment]
Yes I did. I abandoned aeneas. Not what you wanted to hear, but that's it.

**Comment by versae** [comment]
I see. Are you using any other solution that provides satisfactory results?

**Comment by Errorbot1122** [comment]
What other library do you use now?

**Comment by readbeyond** [comment]
@Errorbot1122 here is a list of force aligners: https://github.com/pettarin/forced-alignment-tools

**Comment by readbeyond** [comment]
@ErfolgreichCharismatisch aeneas has a built-in VAD (see the documentation) that can create a speech/silence map before attempting alignment:

- https://www.readbeyond.it/aeneas/docs/vad.html
- Runtime parameters: https://www.readbeyond.it/aeneas/docs/runtimeconfiguration.html#aeneas.runtimeconfiguration.RuntimeConfiguration.VAD_EXTEND_SPEECH_INTERVAL_AFTER
- see the ``--help`` of the relevant command tool you are using

Given the current algorithmic approach of aeneas, based on DTW, there is no meaningful notion of "confidence" that can be emitted. That is a property of forced aligners based on speech recognition.
